packages/flutter/lib/src/animation/curves.dart

```diff
@@ -512,6 +511,15 @@ class Curves {
   /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_decelerate.mp4}
   static const Curve decelerate = _DecelerateCurve._();
 
+  /// A curve that is very steep and linear at the beginning, but quickly flattens out
+  /// and very slowly eases in.
+  ///
+  /// By default is the curve used to animate pages on iOS back to their original
+  /// position if a swipe gesture is ended midway through a swipe.
+  ///
+  /// {http://cubic-bezier.com/#.22,.99,.08,1}
+  static const Cubic fastLinearToSlowEaseIn = Cubic(0.18, 1.0, 0.04, 1.0);
```

Comment:
there's a lot of subjective adjectives here. What each person can mentally visualize this to be can be very different.

Reply:
I'm not sure how to make this easier to understand? I'm also linked the bezier curve so people can see exactly what it looks like by clicking on the link?

Reply:
Not sure how publicly re-usable this is if we can't describe it in an objective way. I'd just leave this privately defined in route.dart.

Reply:
I can totally do this, but I guess I'm unclear on how you're measuring a description's objectivity?